go-vulndb/govulndb.go

```diff
@@ -83,9 +85,14 @@ func (c VulnDB) Update() error {
 		return xerrors.Errorf("index error: %w", err)
 	}
 
+	var modulesErrors []error
 	for moduleName := range modules {
 		entries, err := c.parseModuleEntries(baseURL, moduleName)
 		if err != nil {
+			if strings.HasSuffix(err.Error(), notFoundError) {
```

Comment:
I prefer handling that in `parseModuleEntries`.

Reply:
Done